sonar-python-plugin/src/main/java/org/sonar/plugins/python/PythonPlugin.java

```diff
@@ -20,20 +20,19 @@
 package org.sonar.plugins.python;
 
 import com.google.common.collect.ImmutableList;
+import java.util.List;
 import org.sonar.api.SonarPlugin;
 import org.sonar.api.config.PropertyDefinition;
 import org.sonar.api.resources.Qualifiers;
 import org.sonar.plugins.python.colorizer.PythonColorizer;
 import org.sonar.plugins.python.coverage.PythonCoverageSensor;
 import org.sonar.plugins.python.cpd.PythonCpdMapping;
 import org.sonar.plugins.python.pylint.PylintConfiguration;
+import org.sonar.plugins.python.pylint.PylintImportSensor;
 import org.sonar.plugins.python.pylint.PylintRuleRepository;
 import org.sonar.plugins.python.pylint.PylintSensor;
-import org.sonar.plugins.python.pylint.PylintImportSensor;
 import org.sonar.plugins.python.xunit.PythonXUnitSensor;
 
-import java.util.List;
-
 public class PythonPlugin extends SonarPlugin {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this use of "SonarPlugin"; it is deprecated. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)